lib/pengine/native.c

```diff
@@ -976,15 +976,13 @@ pe__resource_xml(pcmk__output_t *out, va_list args)
     GList *only_node G_GNUC_UNUSED = va_arg(args, GList *);
     GList *only_rsc = va_arg(args, GList *);
 
+    int rc = pcmk_rc_no_output;
```

Comment:
This commit does some rearranging to get variables in the same order that they're set as XML attributes.